src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/HotReload/IProjectHotReloadSessionCallback.cs

```diff
@@ -7,14 +7,10 @@ namespace Microsoft.VisualStudio.ProjectSystem.VS.HotReload;
 
 public interface IProjectHotReloadSessionCallback
 {
-    bool SupportsRestart { get; }
-
     Task OnAfterChangesAppliedAsync(CancellationToken cancellationToken);
 
     Task<bool> StopProjectAsync(CancellationToken cancellationToken);
 
-    Task<bool> RestartProjectAsync(CancellationToken cancellationToken);
-
```

Comment:
There are public API changes, but I don't see any changes to the API `.txt` files.